pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -430,6 +432,9 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 	// With device UUID, zedagent is ready to initialize and activate all subscriptions.
 	initPostOnboardSubs(zedagentCtx)
 
+	// Wait until NodeAgent is ready
+	waitUntilNodeAgentReady(zedagentCtx, stillRunning)
```

Comment:
Could you please add a comment why zedagent depends on NodeAgent and cannot continue until it is available. You have it described in Jira but would be good to have it in the code as well.

Reply:
Ok, I will do it.

Reply:
Done